Dockerfile

```diff
@@ -1,8 +1,30 @@
-# Parent Image
-FROM ubuntu:latest
+# RMG Dockerfile 
+# The parent image is the base image that the Dockerfile builds upon.
+# The RMG installation instructions suggest Anaconda for installation by source, however, we use micromamba for the Docker image due to its smaller size and less overhead.
```

Comment:
That's true but miniconda does not have a well maintained docker image base (I searched around a bit for one) and I have built before using Ubuntu -> install miniconda but then the final image of rmg blows out by >2gb

Also, since our group maintains arc and T3, we build off the rmg dockerfile when creating our dockerfiles. When I use micromamba version of rmg + arc + t3, we get about 4.5gb of a compressed image if I recall correctly (or maybe it's just rmg + arc)

Either way, I still believe that micromamba is better for this dockerfile as it offers a lot smaller image. And I disagree with the argument that we should have a dockerfile exactly like the instructions. Users downloading the image from docker hub will almost certainly not care about the behind the scenes of the docker build. But maybe I am not realizing another use case?

Reply:
The docstring just needs to be update